src/pages/app-blog/app-blog.scss

```diff
@@ -24,98 +29,193 @@
+      border-bottom: 1px solid #9baaab;
+      border-left: 1px solid #9baaab;
+      border-top: 1px solid #9baaab;
+      border-right: none;
+      background-color: #fff;
+      padding: 5px;
+      font-size: 16px;
+      font-family: 'Roboto-Regular', sans-serif;
+    }
 
-      ::placeholder {
-        color: $gray-500;
-      }
+    ::placeholder {
+      color: $gray-500;
+    }
 
-      button {
-        background: #c65300;
-        padding: 12px 15px;
-        box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.27);
-        color: #fff;
+    button {
+      background: #c75300;
+      padding: 14px 15px;
+      box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.27);
+      color: #fff;
+      border: none;
+      border-top-right-radius: 2px;
+      border-bottom-right-radius: 2px;
 
-        @include media-breakpoint-down(sm) {
-          background: #fff;
-          border: solid 1px #c65300;
-          padding: 4px 15px;
-        }
+      @include media-breakpoint-down(sm) {
+        background: #fff;
+        border: 1px solid #c75300;
+        padding: 4px 15px;
+      }
 
-        span {
-          font-size: 14px;
-          text-transform: uppercase;
-          color: $gray-dark;
-          font-weight: bold;
+      span {
+        font-size: 14px;
+        text-transform: capitalize;
+        color: $gray-dark;
+        font-weight: bold;
 
-          i {
-            color: #000;
-          }
+        i {
+          color: #000;
         }
       }
+    }
 
-      .contact-icons-label {
-        margin: 0;
-      }
+    .contact-icons-label {
+      margin: 0;
+      font-weight: 700;
+      letter-spacing: 1.02px;
+      font-size: 16px;
+      color: #9baaab;
+    }
 
-      .contact-icons {
-        width: 100%;
-      }
+    .contact-icons {
+      width: 100%;
+    }
 
-      .contact-icon {
-        margin: 10px;
-        img {
-          width: 20px;
-        }
+    .contact-icon {
+      margin-right: 12px;
+      img {
+        width: 20px;
       }
     }
   }
 }
 
 .blog-filters {
   position: sticky;
-  top: 70px;
   background: #fff !important;
   z-index: 1;
 
   .blog-filters-nav {
     background-color: unset;
 
+    @include media-breakpoint-up(lg) {
+      margin-left: 10%;
+      margin-right: 10%;
+    }
+    @include media-breakpoint-down(lg) {
+      margin: 0;
+    }
+
+    .blog-search-group {
+      @include media-breakpoint-up(xl) {
+        padding-left: 66.6px; // The number of the beast
+      }
+      @include media-breakpoint-down(md) {
+        width: calc(100% - 16px);
+        margin: 8px;
+      }
+      width: 100%;
+    }
+
+    .blog-search-input {
+      padding: 4px 4px 4px calc(2.4rem + 2px);
+      border: none;
+      background-color: #f4f5f9;
+      box-shadow: inset -1px -1px 1px 0 rgba(0, 0, 0, 0.15);
+      border-radius: 3px;
+      width: 100%;
+      height: 32px;
+      font-family: Muli;
+      color: #5d534e;
+      font-size: 13px;
+      font-weight: 300;
+      line-height: 18px;
```

Comment:
Not functionality related, but for longer styles like this example I like putting the styles in alphabetical order. It's easier to view top down when you can view all the border styles together, all the font styles together, etc.